Student/Student/StudentAppDelegate.swift

```diff
@@ -106,11 +112,12 @@ class StudentAppDelegate: UIResponder, UIApplicationDelegate, AppEnvironmentDele
         if
             options[.sourceApplication] as? String == Bundle.teacherBundleID,
             let components = URLComponents(url: url, resolvingAgainstBaseURL: true),
-            components.path == "student_view",
+            components.path.contains("student_view"),
             let fakeStudent = LoginSession.mostRecent(in: .shared, forKey: .fakeStudents) {
             if environment.currentSession != nil {
                 NativeLoginManager.shared().logout() // Cleanup old to prevent token errors
             }
+            self.environment.k5.sessionDefaults?.isK5StudentView = components.path.contains("k5")
```

Comment:
`self` could be omitted and `sessionDefaults` accessed directly via `environment`.